book/content/open-source-comms/comms-channels.md

```diff
@@ -0,0 +1,34 @@
+# Communication Channels
+
+The channel(s) through which you choose to communicate with your community will have a big impact on the experience and sense of belonging for your community members.
+The choice of channels will depend on the specific requirements of your project, such as keeping the community informed of any known bugs or upcoming features, assessing the current needs from the community members, inviting comments on the management plans, facilitating the overall progress of the project, as well as having somewhere to field questions and feedback.
+
+When setting up your communication channels, there are some important things to consider such as the following:
+
+* How easy is it for the community to find these channels and subscribe/unsubscribe to them?
+* How easy is it for members of the community to find and join specific conversations they are interested in?
+* Will it be a "noisy" channel (for example, lots of notifications)?
+* Can users control or filter what kind of information they receive?
+
+There are three commonly used channels: mailing lists (such as [Topicbox](https://www.topicbox.com/), or [Google groups](https://support.google.com/mail/thread/14635045?hl=en)), community forums (such as [Vanilla Forum](https://vanillaforums.com/en/software/) or [Discourse](https://www.discourse.org/)) and Internet Relay Chat (IRC) (such as [Gitter](https://gitter.im/) or [Slack](https://app.slack.com/signin)). 
+Besides these, many communities also use platforms for weekly or monthly newsletters, blogs, and social media on [Twitter](https://twitter.com/) or [Facebook](https://www.facebook.com/).
+
+In the table below, some properties of the first three communication platforms have been highlighted, which will allow Open Source project leaders to choose the most appropriate channels for their communities.
+
+## Features of different Communication Channels
+
+| Feature | Mailing Lists | Forums | Gitter |
+| :--- | :---: | :---: | :---: |
+| Media sharing | ✅ (cumbersome) | ✅ | ✅ (cumbersome) |
+| Archiveable | ✅ | ✅ | |
+| Persistant | ✅ (up to a limit) | ✅ | |
+| Noisy | ✅ | | ✅ |
+| Open | | ✅ | ✅ |
+| Consistent UX* | | ✅ | ✅ |
+| Threading/Filtering of conversations | | ✅ | ✅ |
+| Social sharing | | ✅ | ✅ |
+| User profiles | | ✅ | ✅ |
+| "Friending"/"Ignoring" capabilities | | ✅ | |
+| Searchable by topic | | ✅ | |
```

Comment:
I think it would be a good idea to write two sentences to each row here to explain what you mean and give a bit more context. Several of the rows in this table are quite subtle. With only two-three words labelling a row and a yes/no tick my reaction to disagreeing with the table is "the author must have no experience with column X which is why they are so uninformed".

An alternative would be to pick stand in tools for each category and compare those. That way you avoid the problem that everyone knows a love (discourse) and a hate (phpBB) example (in this case forum software). There is a huge amount of variation amongst the tools in each category, sometimes even more than between the categories :-/